xdate/0.8/pom.xml

```diff
@@ -42,7 +42,7 @@
     </license>
 
     <license>
-      <name>GPL</name>
+      <name>GPL-2.0-or-later</name>
```

Comment:
The License at https://github.com/arshaw/xdate/blob/v0.8/GPL-LICENSE.txt does not have the "or later" clause so I think this should be `GPL-2.0-only`?

Reply:
@siepkes It appears we are both right and we're both wrong:
https://www.fsf.org/blogs/rms/rms-article-for-claritys-sake-please-dont-say-licensed-under-gnu-gpl-2